docs/quickstart.rst

```diff
@@ -243,7 +237,7 @@ Nesting Fields
 
 .. note::
 
-    By default, webargs only parses nested fields using the ``json`` request location. You can, however, :ref:`implement your own parser <custom-parsers>` to add nested field functionality to the other locations.
+    Of the default supported locations in webargs, only the ``json`` request location supports nested datastrctures. You can, however, :ref:`implement your own data loader <custom-loaders>` to add nested field functionality to the other locations.
```

Comment:
s/datastrctures/datastructures/

Reply:
Good catch. Let me patch this right now so I don't forget.